app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt

```diff
@@ -20,9 +21,10 @@ class KeyboardHelper {
     }
 
     /**
-     * This method shows softinputkeybord
-     * @param view is the input view
-     * @param context context of the activity
+     * This method shows the soft input keyboard.
+     *
```

Comment:
remove this extra line

Reply:
This was suggested by @BenHenning previously